README.md

```diff
@@ -16,10 +16,14 @@ by setting `:upload.:port` configuration value.
 
 #### Upload
 
+Generate some sample data : 
+
+`seq 1 100000 | while read i ; do echo "line $1 : foo bar baz basho" >> /tmp/upload.garbage.txt ; done`
+
 You can upload files via curl:
 
 ```bash
-$ curl -X PUT http://localhost:8080/uploads/your-file-name --data-binary @path/to/your/file
+curl -X PUT http://localhost:8080/uploads/garbage.txt --data-binary @/tmp/upload.garbage.txt
```

Comment:
Could you change the path for GET request below as well?

Reply:
done